mod/groups/views/default/forms/groups/find.php

```diff
@@ -3,14 +3,17 @@
  * Group tag-based search form body
  */
 
-$tag_string = elgg_echo('groups:search:tags');
-
-$params = array(
+echo elgg_view_field([
```

Comment:
I wonder how we could make this work with a postfix on the input. I see we have several use cases where it would come handy. Food for thought.